### PR TITLE
Reports and customer management related reports

### PR DESCRIPTION
**Pull request by rajupaladiya**
Reports and customer management related reports #49

### CODE REVIEW

**Comment by DmitriyKulagin** [comment]
Hi @rajupaladiya,
Unfortunately, we cannot approve your pull request, as it contains the commits of other participants, and they will be re-written on your behalf. You also added this commit (https://github.com/nopSolutions/nopCommerce-Docs/pull/51/commits/a403ae0fa2113bbf6f0df4acbd2e53ed1025f803) on your behalf, which was previously added to the "master" branch (https://github.com/nopSolutions/nopCommerce-Docs/commit/96c6dce0fed44f3fb4ba1fd5f303ea08239140ff). This is not correct with respect to the author of this commit. You need to make a pull request so that it contains only your corrections.

File: user-guide/running/product-management/products/adding-products/index.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ In the **Prices** section define:
 - **Unit of product** - measure of a previously entered value.
 - **Reference amount** - the base amount.
 - **Reference unit** - measure of a previously entered value.
-- **Discounts**. Learn how to set up discounts [here](xref:/user-guide/marketing/promotional/discounts/index.html).
```
Comment (DmitriyKulagin) [inline user-guide/running/product-management/products/adding-products/index.md]: I still see that commits from the master branch are being overwritten.

**Comment by DmitriyKulagin** [comment]
Hi, @rajupaladiya 
I slightly adjusted your pull request. You did a great job, thanks!

**Comment by rajupaladiya** [comment]
@DmitriyKulagin - Thank you Dimitry for this.
I will resume my contribution to documentation from next week.

**Pull request merged**